code/controllers/subsystem/mapping.dm

```diff
@@ -396,6 +407,9 @@ GLOBAL_LIST_EMPTY(the_station_areas)
 		if (VM.config_max_users > 0 && players > VM.config_max_users)
 			mapvotes.Remove(map)
 			continue
+		if(previous_maps && previous_maps[VM.map_name] > 7)
```

Comment:
previous maps is always defined, meant to check length()?

Reply:
Does byond take a shit on looking up dictionary keys that aren't actually in the dictionary?